code/modules/unit_tests/unit_test.dm

```diff
@@ -146,8 +149,11 @@ GLOBAL_VAR_INIT(focused_tests, focused_tests())
 
 	log_world("::[priority] file=[file],line=[line],title=[map_name]: [type]::[annotation_text]")
 
-/proc/RunUnitTest(test_path, list/test_results)
-	if (ispath(test_path, /datum/unit_test/focus_only))
+/proc/RunUnitTest(datum/unit_test/test_path as anything, list/test_results)
```

Comment:
```suggestion
/proc/RunUnitTest(datum/unit_test/test_path, list/test_results)
```

Reply:
I am sure that it's needed.

Reply:
It is absolutely not needed, this does absolutely nothing in proc arguments. It only does something in loops

Reply:
I think that I must have been seeing a fluke before. For instance I have to close and re-open VSC to get the tests to update with new code sometimes... Sorry about this confusing addition. I have removed it now.